frontend/src/compliance_reporting/components/ComplianceReportingTable.js

```diff
@@ -33,7 +32,45 @@ const ComplianceReportingTable = (props) => {
     id: 'type',
     minWidth: 75
   }, {
-    accessor: item => <ComplianceReportStatus status={item.status} />,
+    accessor: (item) => {
```

Comment:
Should this be factored out into a helper?

Reply:
Sure